libs/cart/driver/magento/src/models/responses/cart-item.ts

```diff
@@ -2,14 +2,27 @@ import {
   MagentoMoney,
   MagentoDiscount,
 } from '@daffodil/driver/magento';
-import { MagentoProduct } from '@daffodil/product/driver/magento';
+import { MagentoProductStockStatusEnum } from '@daffodil/product/driver/magento';
 
 export enum MagentoCartItemTypeEnum {
 	Simple = 'SimpleCartItem',
 	Bundle = 'BundleCartItem',
 	Configurable = 'ConfigurableCartItem'
 }
 
+export interface MagentoCartItemProduct {
+	__typename: string;
```

Comment:
Can you link where each of these props is used in a transformer? I'm not certain `stock_status` is for example.

Reply:
https://github.com/graycoreio/daffodil/blob/06390525919ab12b3ca77b64401b0f641ba0874f/libs/cart/driver/magento/src/transforms/outputs/cart-item/simple-cart-item-transformer.ts#L32

Reply:
https://github.com/graycoreio/daffodil/blob/06390525919ab12b3ca77b64401b0f641ba0874f/libs/cart/driver/magento/src/transforms/outputs/cart-item/simple-cart-item-transformer.ts#L21

Reply:
https://github.com/graycoreio/daffodil/blob/06390525919ab12b3ca77b64401b0f641ba0874f/libs/cart/driver/magento/src/transforms/outputs/cart-item/simple-cart-item-transformer.ts#L22

Reply:
https://github.com/graycoreio/daffodil/blob/06390525919ab12b3ca77b64401b0f641ba0874f/libs/cart/driver/magento/src/transforms/outputs/cart-item/simple-cart-item-transformer.ts#L26

Reply:
https://github.com/graycoreio/daffodil/blob/06390525919ab12b3ca77b64401b0f641ba0874f/libs/cart/driver/magento/src/transforms/outputs/cart-item/simple-cart-item-transformer.ts#L28

Reply:
Indeed.